samples/spring/src/main/kotlin/io/github/rybalkinsd/kohttp/spring_kohttp/Controller.kt

```diff
@@ -0,0 +1,37 @@
+package io.github.rybalkinsd.kohttp.spring_kohttp
+
+import io.github.rybalkinsd.kohttp.dsl.httpGet
+import io.github.rybalkinsd.kohttp.dsl.httpPost
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
```

Comment:
If you're interested in exploring other options of API declaration, I'd suggest to have a look at MVC dsl, https://spring.io/blog/2019/04/03/spring-tips-webmvc-fn-the-functional-dsl-for-spring-mvc

We can have examples with two approaches

Reply:
Yep, I can add an example using `Webflux` 👍